tasks/pre_checks/001_validate_network_interfaces.yml

```diff
@@ -70,6 +70,37 @@
         reject('skipped') | map(attribute='bond_item.ansible_facts.otopi_net_host') | list }}
     register: otopi_host_net
   - debug: var=otopi_host_net
+  - name: Collect interfaces type
+    shell: nmcli con show {{ nic }} | grep connection.type
+    with_items:
+      - "{{ otopi_host_net }}"
+    loop_control:
+      loop_var: nic
+    register: interfaces_type
+  - debug: var=interfaces_type
+  - name: Check if there are team devices
+    set_fact:
+      is_team: "{{ nic_if.stdout.find('team') > 0 }}"
+    when: nic_if.stdout.find('team') != -1
+    with_items:
+      - "{{ interfaces_type.results }}"
+    loop_control:
+      loop_var: nic_if
+    register: team_list
+  - debug: var=team_list
+  - name: Get list of team devices
+    set_fact:
+      team_if: "{{  team_list.results | reject('skipped') | map(attribute='nic_if.nic') | list }}"
+  - debug: var=team_if
+  - name: Filter unsupported interfaces type
+    set_fact:
+      otopi_host_net: "{{ otopi_host_net | difference(team_if) }}"
+    register: otopi_host_net
+  - debug: var=otopi_host_net
+  - name: Failed if only teaming devices are availible
+    fail:
+      msg: Networking teaming is not supported
```

Comment:
unclear error message. Maybe : `Only Team devices are present. Teaming is not supported`

And add the list of network devices you collected to the error message.